CMakeLists.txt

```diff
@@ -1792,7 +1815,9 @@ add_subdirectory(init)
 ################################################################################
 # Add components
 ################################################################################
-add_subdirectory(components)
+if(HPX_WITH_DISTRIBUTED_RUNTIME)
+  add_subdirectory(components)
```

Comment:
Why can't we have components in a non distributed world?

Reply:
Because that's what classes are for. `HPX_WITH_NETWORKING=OFF` gives you components without turning networking if that's what you're looking for.

Reply:
And to clarify even further, there are actually components/plugins which would be useful also locally: the counters components. But since they still at the moment require components and actions they will stay disabled.